src/agoradatatools/great_expectations/gx/json_schemas/immunohisto/points.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "http://example.com/example.json",
+    "type": "array",
+    "default": [],
+    "title": "Points Schema",
+    "items": {
+        "type": "object",
+        "title": "A Schema",
+        "required": [
+            "genotype",
+            "measurement",
+            "sex"
+        ],
+        "properties": {
+            "genotype": {
```

Comment:
We should also be able to generate a finite set of values for genotype from the data. The two datasets might have some non-overlapping values, but I'm not sure.

Reply:
Where could one find such a list?